apps/web/src/pages/projects/index.tsx

```diff
@@ -1,44 +1,166 @@
 import useSWR from "swr";
 import { PageLayout } from "@/components/ui/page-layout";
-import { columns } from "../table/columns";
-import { DataTable } from "../table/data-table";
 import { Button } from "@/components/ui/button";
 import Link from "next/link";
 import { Plus } from "lucide-react";
 import React from "react";
-import { useRouter } from "next/router";
+import { ListHeading, Paragraph } from "@/components/ui/typography";
+import usePagination from "@/hooks/pagination";
+import { cn } from "@/lib/utils";
 
 export default function Projects() {
-  const { data, isLoading } = useSWR(`/api/openstad/api/project?includeConfig=1`);
+  const { data, isLoading } = useSWR("/api/openstad/api/project?includeConfig=1");
 
-  const router = useRouter();
   if (!data) return null;
 
+  const {
+    page,
+    next,
+    previous,
+    goToPage,
+    hasNext,
+    hasPrevious,
+    pages,
+    goToFirst,
+    goToLast,
+  } = usePagination();
+
+  const meta = {
+    total: 3,
+    size: 4,
+    page: 2,
+    results: 3,
+  };
+
   return (
     <div>
-      <PageLayout
-        pageHeader="Projecten"
-        breadcrumbs={[
-          {
-            name: "Projecten",
-            url: "/projects",
-          },
-        ]}
-        action={
-          <Link href="/projects/create">
-            <Button variant="default">
-              <Plus size="20" />
-              Project toevoegen
-            </Button>
-          </Link>
-        }
-      >
-        <div className="container">
-          <DataTable columns={columns} data={data} onRowClick={(d) =>{
-            router.push(`${router.asPath}/${d.original.id}/widgets`)}
-            }/>
+          <PageLayout
+          pageHeader="Projects"
+          breadcrumbs={[
+            {
+              name: 'Projects',
+              url: '/projects',
+            },
+          ]}
+          action={
+            <Link href="/projects/create">
+              <Button variant="default">
+                <Plus size="20" />
+                Project toevoegen
+              </Button>
+            </Link>
+          }
+          >
+            <div className="container mx-auto py-10">
+              <div className="mt-4 grid grid-cols-2 md:grid-cols-12 items-center py-2 px-2 border-b border-border overflow-x-scroll">
+                <ListHeading className="hidden md:flex">
```

Comment:
Ik zie heel veel herhaling met  
```
<ListHeading className="hidden md:flex">
                  Issues
                </ListHeading>
```
                
                Naar mijn idee zou je dit heel makkelijk in een soort column array kunnen zetten ['Projectnaam', 'data', 'etc'].map(header =>   <ListHeading className="hidden md:flex">
                  {header}
                </ListHeading>)